tools/kagami/src/main.rs

```diff
@@ -363,43 +363,120 @@ mod genesis {
+                    private_key: Some(PrivateKey::from_hex(
+                        Algorithm::Ed25519,
+                        "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+                    )?),
+                    ..ConfigurationProxy::default()
+                }
+                .build()?;
+                writeln!(writer, "{}", serde_json::to_string_pretty(&config)?)
+                    .wrap_err("Failed to write.")
+            }
+        }
+    }
+
+    mod peer {
+        use iroha_config::{
+            genesis::ConfigurationProxy as GenesisConfigurationProxy,
+            iroha::ConfigurationProxy as IrohaConfigurationProxy,
+            torii::{
+                uri::DEFAULT_API_URL, ConfigurationProxy as ToriiConfigurationProxy,
+                DEFAULT_TORII_P2P_ADDR, DEFAULT_TORII_TELEMETRY_URL,
+            },
+        };
+
+        use super::*;
+
+        #[derive(StructOpt, Debug, Clone, Copy)]
+        pub struct Args;
+
+        impl<T: Write> RunArgs<T> for Args {
+            fn run(self, writer: &mut BufWriter<T>) -> Outcome {
+                let config = IrohaConfigurationProxy {
+                    public_key: Some(PublicKey::from_str(
+                        "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
+                    )?),
+                    private_key: Some(PrivateKey::from_hex(
+                        Algorithm::Ed25519,
+                        "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+                    )?),
+                    torii: Some(ToriiConfigurationProxy {
+                        p2p_addr: Some(DEFAULT_TORII_P2P_ADDR.to_owned()),
+                        telemetry_url: Some(DEFAULT_TORII_TELEMETRY_URL.to_owned()),
+                        api_url: Some(DEFAULT_API_URL.to_owned()),
+                        ..ToriiConfigurationProxy::default()
+                    }),
+                    genesis: GenesisConfigurationProxy {
+                        account_public_key: PublicKey::from_str(
+                            "ed01203f4e3e98571b55514edc5ccf7e53ca7509d89b2868e62921180a6f57c2f4e255",
+                        )?.into(),
+                        account_private_key: Some(Some(PrivateKey::from_hex(
+                            Algorithm::Ed25519,
+                            "038ae16b219da35aa036335ed0a43c28a2cc737150112c78a7b8034b9d99c9023f4e3e98571b55514edc5ccf7e53ca7509d89b2868e62921180a6f57c2f4e255"
+                        )?)),
+                        ..GenesisConfigurationProxy::default()
+                    }.into(),
+                    ..IrohaConfigurationProxy::default()
+                }
+                .build()?;
+                writeln!(writer, "{}", serde_json::to_string_pretty(&config)?)
+                    .wrap_err("Failed to write.")
```

Comment:
Same as @ilchu has suggested in his [comment](https://github.com/hyperledger/iroha/pull/3028/files#r1055093571)